tools/clusters-checker/pkg/checkers/process_indices_with_timestamp.go

```diff
@@ -0,0 +1,197 @@
+package checkers
+
+import (
+	"encoding/json"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+)
+
+const (
+	checkAccumulateInterval = 10
+)
+
+var (
+	log = logger.GetOrCreate("checkers")
+)
+
+type clusterChecker struct {
+	missingFromSource      map[string]json.RawMessage
+	missingFromDestination map[string]json.RawMessage
+
+	clientSource         ESClient
+	clientDestination    ESClient
+	indicesNoTimestamp   []string
+	indicesWithTimestamp []string
+
+	startTimestamp, stopTimestamp int
+
+	logPrefix string
```

Comment:
You can use LoggerWithIdentifier for the same result but a cleaner implementation.

Reply:
In order to can use `LoggerWithIdentifier`  I have to include github.com/ElrondNetwork/elrond-eth-bridge
Will let as it is for the moment